lib/converterbase.rb

```diff
@@ -91,7 +91,15 @@ def rstrip_all_lines(data)
   #
   def convert_numbers(data)
     # 小数点を・に
-    data.gsub!(/([\d０-９#{KANJI_NUM}]+?)[\.．]([\d０-９#{KANJI_NUM}]+?)/, "\\1・\\2")
+    data.gsub!(/([\d０-９#{KANJI_NUM}]+?)[\.．]([\d０-９#{KANJI_NUM}]+?)/) do |match|
+      integer = $1
+      decimal = $2
+      if [/\d/, /[０-９]/, /[#{KANJI_NUM}]/].any? {|r| integer[-1] =~ r && decimal[0] =~ r }
```

Comment:
__[rubocop]__
* Style/SpaceInsideBlockBraces: Space between { and | missing.

<!-- Comment by [SideCI](https://sideci.com) -->